### DISCUSSION TITLE
Less page views after proxying Plausible in Next.js

### DISCUSSION THREAD

**Discussion by webdevelukas** [plausible/analytics#1110] [Anything Else]
Hi all 👋 

Cause i was wondering that the numbers of unique visitors (280) and page views (15.6k) last month seem to be unrealistic, I followed this [guide](https://plausible.io/docs/proxy/guides/nextjs) to proxy Plausible analytics in Next.js.

![image](https://user-images.githubusercontent.com/54030139/120980359-d47d9380-c776-11eb-8c60-c04adbbd8731.png)

My thoughts: When I proxy the script, less visitors get blocked by ad blockers. So the numbers of unique visitors should rise and maybe they getting closer to the counts of page views. 

After one more week now the numbers seem to be realistic now:
![image](https://user-images.githubusercontent.com/54030139/120979407-df83f400-c775-11eb-894e-7a937ac7ea91.png)

But the effect is different to my thoughts. The numbers of the unique visitors are staying the same, but the count of page views is decreased.

It would be great if someone could explain me what I am missing here. I really like to unterstand what I am doing there. :-)

Thanks, Lukas ✌️ 

The Plausible analytics dashboard of the project: https://plausible.io/wemake.de
The project itself on Github: https://github.com/webdevelukas/wemake

**Comment by ukutaht** [comment]
Hey!

It looks to me like your dashboard has some bot traffic which has recently subsided. 280 visitors and 15k pageviews is pretty unrealistic.

It's possible that since you're proxying through Vercel, they are doing a better job at filtering bots out than we are.

**Reply by webdevelukas** [reply]
Alright, thanks for fast reply. :-)

**Comment by webdevelukas** [comment]
Hi @ukutaht,

since the last time I reported these heavy numbers of page views by less unique visitors everything seems to run normal. But two weeks ago there where only two days where it happened again. I don't know how to work with these numbers. Is there something like a bot "attack" against vercel or the page then, or is it something plausible related?

https://plausible.io/wemake.de?date=2021-07-21
https://plausible.io/wemake.de?date=2021-07-24

**Reply by ukutaht** [reply]
It's hard to tell. Clearly these are not regular humans clicking around the page. On some sessions I can see more than 160 pageviews per second which is unrealistic for humans. Some are using Safari on Mac and others Safari on iOS although it's possible to spoof the user-agent as well.

It's not a crawler either because the large number of pageviews is on the home page `/` not on any other page.

I can identify and delete the sessions from the backend but I'm not sure how much help it will be if it keeps happening.

**Reply by webdevelukas** [reply]
Thanks for the fast response. 👍 

It would be cool if you can delete it, but surely I want to solve this problem long term. Could you please give me a hint what to do next? Should I contact vercel then?

**Reply by ukutaht** [reply]
Do you have access logs from Vercel?

You could maybe plug them into a tool like AWStats/Goaccess figure out which IP addresses are behind these traffic spikes. I wish we could do this for you but we don't log IP addresses on our side for privacy+legal reasons. All I know is that when there are spikes, it's all coming from a single IP address because it's interpreted as a single unique user in Plausible.

You could also post the access logs here or send them to us privately for analysis. I'm interested in this to improve bot detection on our side. Ideally we should be able to ignore bot traffic like this to reduce noise.

**Reply by webdevelukas** [reply]
I've sent a request to the vercel support for the access logs. I hope that i can send you something you can work with.

**Reply by ukutaht** [reply]
Thanks, hopefully they get back!
